src/js/components/DateInput/index.d.ts

```diff
@@ -17,6 +17,7 @@ export interface DateInputProps {
   name?: string;
   onChange?: (event: { value: string | string[] }) => void;
   value?: string | string[];
+  disabled?: boolean;
```

Comment:
I think we might be missing the underlying JSX.IntrinsicElements, since DateInput should be using `JSX.IntrinsicElements['input']` then `disabled` should already be included as part of it. Could you please take a look at whether it is hiding somewhere or we should add the JSX.IntrinsicElements['input'] manually?

Reply:
Thanks Shimi! I've added the JSX.IntrinsicElements['input'] here https://github.com/grommet/grommet/pull/5165/commits/f4a7f51ce21b2de562e55c5e47b70b4ad55abf71

Reply:
Great! I'm guessing that line #20 is no longer needed if we are using the `JSX.IntrinsicElements['input']` now, since `disabled` is a prop of `JSX.IntrinsicElements['input']` by default, does that work?

Reply:
@jaygiang this should be fine once you remove the `disabled?: boolean;`

Reply:
@britt6612 @ShimiSun Thanks for your help! I have removed `disabled?: boolean;`, but I'm not sure why it's still not passing.